### DISCUSSION TITLE
icon display style have problem

### DISCUSSION THREAD

**Discussion by stoneLee81** [singerdmx/flutter-quill#174] [Web]
![image](https://user-images.githubusercontent.com/18001708/114732410-11c83500-9d75-11eb-835c-42d79939e314.png)
There is a problem with the display style, under flutter web

**Comment by gaganyadav80** [comment]
Is it on the web or mobile?
Also share the code you are using. 
And if you see any error of overflow or something also do share them.

If you can then provide the steps to reproduce. :)

**Comment by stoneLee81** [comment]
I executed flutter-quill demo directly under the web; The os font is chinese, flutter Channel stable, 2.0.4

**Comment by gaganyadav80** [comment]
Hey! I know I'm replying too late but I hope this will help you. So I ran the example code and had the same issue as yours. 

I haven't checked out the problem yet because I'm ill for the past 10 days or so. But I have a workaround for now in case you have this problem in your own project also.

If you go to `home_page.dart` file and comment the lines 161-162 which is container padding then you'll see that the buttons will be visible properly.
The workaround is that you don't add padding or margin around toolbar.
